transaction.go

```diff
@@ -28,11 +28,11 @@ import (
 
 	"github.com/corazawaf/coraza/v3/bodyprocessors"
 	"github.com/corazawaf/coraza/v3/collection"
+	utils "github.com/corazawaf/coraza/v3/internal/strings"
+	url2 "github.com/corazawaf/coraza/v3/internal/utils/url"
```

Comment:
Here. Probably we need to decide if we want `internal/url` or `internal/utils/url`?